app/models/potion.rb

```diff
@@ -16,15 +16,19 @@ class Potion < ApplicationRecord
 
   DUPLICATE_MATCH = 'is a duplicate of a unique in-game item'
 
+  def canonical_model
+    canonical_potion
+  end
+
   def canonical_models
-    return Canonical::Potion.where(id: canonical_potion_id) if canonical_potion.present?
+    return Canonical::Potion.where(id: canonical_potion_id) if canonical_model_matches?
 
-    matching = Canonical::Potion.where('name ILIKE ?', name)
-    matching = matching.where('magical_effects ILIKE ?', magical_effects) if magical_effects.present?
+    canonicals = Canonical::Potion.where('name ILIKE ?', name)
```

Comment:
Change variable name for consistency with this method in other in-game item classes.